src/interfaces/slack_bot/src/constants.ts

```diff
@@ -0,0 +1,151 @@
+  INVALID_SUMMARIZE_FILE_TYPE: `Sorry, I can't summarize that file type. I can summarize PDFs, Word documents (doc and docx), and text files.`,
+  INVALID_RAG_CHAT_FILE_TYPE: `Sorry, I can't read that file type. Currently I can only read PDFs.`,
+  INVALID_TEMPERATURE:
+    'The value you provided is not a valid temperature (0.0-5.0). Please try again.',
+  USER_NOT_IN_CHANNEL:
+    'Looks like the conversation linked is from a private channel. Summaries can only be generated for public channels or private channels you are a member of.',
+};
+export const SLACK_API_ERRORS: { [key: string]: string } = {
+  not_in_channel:
+    "Looks like I'm not in the channel you specified. Please invite me to the channel and try again.",
+  channel_not_found:
+    "Looks like I cannot find the channel you specified. If it's a private channel, please invite me to the channel and try again.",
+  thread_not_found: 'Oops, I cannot find the thread you specified.',
+  access_denied: 'Sorry, I do not have the required access permissions.',
+};
+export const STOP_REPLYING_MESSAGE = 'I will stop responding :speak_no_evil:';
+export const HELP_MESSAGE = `Hi! I'm Command, Cohere's conversational AI in Slack! :wave:
+
+You can chat with me to learn new subjects through open and engaging conversations. You can also summarize a thread that you missed out on with a built in shortcut.
+
+
+\`\`\`Slash Commands\`\`\`
+
+*:orange_book: Feature and help commands*:
+
+\`/command-help\` - Show this help message
+\`/summarize [thread link]\` - Post a summary of a slack thread given a link
+
+*:gear: Configuration commands*:
+
+\`/set-model [model name]\` - Set the model used for a specific channel
+\`/view-model\` - View the model used for a specific channel
+\`/set-temperature [0.0 - 5.0]\` - Set the temperature used for a specific channel
+\`/view-temperature\` - View the temperature used for a specific channel
+\`/set-preamble [preamble]\` - Set the preamble override used for a specific channel
+\`/view-preamble\` - View the preamble override used for a specific channel
+\`/set-tools\` - Set tools for a specific channel
+\`/view-tools [all]\` - View all tools available in your Cohere account or tools enabled for a specific channel
+\`/setup-command\` - _Admins only_ - Setup Command for the workspace
+
+
+\`\`\`Context Menu Options\`\`\`
+\`Summarize thread\` - Post a summary of a thread with one simple click
+
+
+*:question: What does that reaction mean?*
+
+:shushing_face: - To indicate Command should stop responding in the current thread.
+:warning: - An error occurred while processing the specific message that has this reaction. The message sender will be provided with more information regarding the error in an ephemeral message.
+
+
+For further support, feel free to reach out to <mailto:coralbot@cohere.com?Subject=Command%20Slackbot%20Support|coralbot@cohere.com>.
```

Comment:
Can you change these emails?

Reply:
It has been changed to support@cohere.com.